### PR TITLE
[DPE-1180] Replace icon with the one provided by PM/WEB design team

### PR DESCRIPTION
**Pull request by taurus-forever**
It is a first repo to update the icon, check it in stable, and copy to all other Data Platform charms.

### CODE REVIEW

**Review by taurus-forever — COMMENTED** [review]
I am worried about 
1) the size, the old icon 1Kb, the new one 19,5Kb.
2) I see header `<?xml version="1.0" encoding="UTF-8" standalone="no"?>` inside SVG icon. Is it expected here?

File: icon.svg [hunk 1]
```diff
@@ -1 +1,288 @@
...
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (taurus-forever) [inline icon.svg:1]: Is XML header expected in SVG icon?

Comment (taurus-forever) [reply icon.svg:1]: Tnx! Any ideas why XML header was missing for old SVG file?

Comment (dragomirp) [reply icon.svg:1]: Likely removed to reduce size (minification).

**Review by delgod — APPROVED** [review]

File: icon.svg (hunk 1 above)
Comment (delgod) [reply icon.svg:1]: yes, we need vector

**Review by deusebio — APPROVED** [review]
LGTM

**Comment by taurus-forever** [comment]
Aborting as we need a [vector](https://github.com/canonical/data-platform-libs/pull/54#discussion_r1153996847) SVG icon.

**Pull request closed without merging**